makefile

```diff
@@ -76,26 +76,5 @@ report: all
 	dscanner --report src > src/dscanner-report.json
 	sonar-runner
 
-.ONESHELL:
```

Comment:
`ONESHELL` is only available in [Make 3.82](https://lists.gnu.org/archive/html/info-gnu/2010-07/msg00023.html).
Travis uses 3.81 and there's no easy way to upgrade.